test/connectors/TaxEnrolmentConnectorSpec.scala

```diff
@@ -40,7 +40,10 @@ class TaxEnrolmentConnectorSpec extends ConnectorSpecHelper {
       )
   }
 
-  ".enrolSubscriber" should {
+  ".enrolSubscriber 4MLD" should {
```

Comment:
Missing specs for 5MLD?

Reply:
Updated but getting failed tests when mocking config returns